src/saltext/vmware/states/vmc_dhcp_profiles.py

```diff
@@ -0,0 +1,420 @@
+
+    sddc_id
+        The Id of SDDC for which the DHCP profile should be added
+
+    type
+        The type of DHCP profile for which the given dhcp belongs to. Possible values: server, relay
+
+    dhcp_profile_id
+        Id of the DHCP profile to be added to SDDC
+
+    verify_ssl
+        (Optional) Option to enable/disable SSL verification. Enabled by default.
+        If set to False, the certificate validation is skipped.
+
+    cert
+        (Optional) Path to the SSL client certificate file to connect to VMC Cloud Console.
+        The certificate can be retrieved from browser.
+
+    server_addresses
+        when type is relay, this field indicates DHCP relay addresses(DHCP server IP addresses
+        for DHCP relay configuration).
+        Both IPv4 and IPv6 addresses are supported.
+
+        when type is server, this field indicates DHCP server address in CIDR format.
+        Both IPv4 and IPv6 address families are supported.
+        Prefix length should be less than or equal to 30 for IPv4 address family
+        and less than or equal to 126 for IPv6.
+        When not specified, IPv4 value is auto-assigned to 100.96.0.1/30.
+
+        Note: This field is optional only when type is server and
+        is mandatory when type is relay.
+
+    tags
+        (Optional) Opaque identifiers meaningful to the user.
+
+        .. code::
+
+            tags='[
+                {
+                    "tag": "<tag-key-1>"
+                    "scope": "<tag-value-1>"
+                },
+                {
+                    "tag": "<tag-key-2>"
+                    "scope": "<tag-value-2>"
+                }
+            ]'
+
+    lease_time
+        (Optional) IP address lease time in seconds. Minimum value is 60.
+        Maximum value is 4294967295. Default value is 86400
+        Note: This field is applicable only when type is server
+
+    display_name
+        Identifier to use when displaying entity in logs or GUI.
+
+    Example values:
+
+        .. code::
+
+            for dhcp-server-profiles:
+            {
+                "display_name": "dhcp-test",
+                "server_addresses": [
+                    "10.22.12.2/23"
+                ],
+                "tags": [
+                    {
+                        "tag": "tag1",
+                        "scope": "scope1"
+                    }
+                ],
+                "lease_time": 86400
+            }
+
+            for dhcp-relay-profiles:
+            {
+                "display_name": "dhcp-test",
+                "server_addresses": [
+                    "10.1.1.1"
+                ],
+                "tags": [
+                    {
+                        "tag": "tag1",
+                        "scope": "scope1"
+                    }
+                ]
+            }
+
+    """
+
+    input_dict = {
+        "server_addresses": server_addresses,
+        "tags": tags,
+        "lease_time": lease_time,
+        "display_name": display_name,
```

Comment:
this would be best as `display_name or dhcp_profile_id`, I think? Or is that the default behavior already?

Reply:
this is the default behavior.  display_name indicates the name of the dhcp profile

Reply:
will make the documentation changes as mentioned in the below comment
https://github.com/saltstack/salt-ext-modules-vmware/pull/84#discussion_r728153576

after documentation change, this will be clarified